packages/utils/messages.ts

```diff
@@ -179,8 +196,28 @@ export const typesRegistry = new Registry([
   // Custom types not in @cosmjs/stargate default registry.
   ...([
     ['/cosmos.slashing.v1beta1.MsgUnjail', MsgUnjail],
+    ['/cosmos.authz.v1beta1.MsgGrant', MsgGrant],
+    ['/cosmos.authz.v1beta1.MsgExec', MsgExec],
     ['/cosmos.authz.v1beta1.GenericAuthorization', GenericAuthorization],
+    ['/cosmos.authz.v1beta1.Grant', Grant],
```

Comment:
<img width="589" alt="Screenshot 2023-03-23 at 11 43 04 PM" src="https://user-images.githubusercontent.com/6721426/227445327-88d83e60-53e0-45cb-84b6-838b72a4734e.png">

I believe the 3 `authz` messages you added are included by default in the registry types from line 193. only `GenericAuthorization` needs to be manually added here from `authz`